.github/workflows/build_devcontainer.yml

```diff
@@ -0,0 +1,29 @@
+name: "build-devcontainer"
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+    paths:
+      - .devcontainer/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The devcontainers CI docs doesn't specify that we should login manually before executing their action, is this because of a bug on their part ?